tests/core/tests/views.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+from django.test.testcases import TestCase
+from django.core.urlresolvers import reverse
+from core.models import Category
+
+
+class ExportViewMixinTest(TestCase):
+
+    def setUp(self):
+        self.url = reverse('export-category')
+        self.cat1 = Category.objects.create(name='Cat 1')
+        self.cat2 = Category.objects.create(name='Cat 2')
+
+    def test_get(self):
+        response = self.client.get(self.url)
+        self.assertContains(response, self.cat1.name, status_code=200)
+        self.assertTrue(response['Content-Type'], 'text/html')
+
+    def test_post(self):
+        data = {
+            'file_format': '0',
```

Comment:
0 looks weird for a file format, here is an idea that needs review: https://github.com/django-import-export/django-import-export/pull/77

Reply:
Personally it doesn't disturb me.
I would prefer `constant.CSV`.

Reply:
agree that it would be nicer to have descriptive string

Reply:
I don't think, it should be part of this PR. I tried to do non breaking core and PR #77 modifies more than my goal.